packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/oktatookta/appatidp.md

```diff
@@ -12,6 +12,6 @@ When you are configuring federation between two Okta orgs using OpenID Connect:
     * Needs to be a secure domain that you own
     * Should match the redirect URI sent in the authorize request from the client
 
-1. Assign a group or leave the **Everyone** default. Be sure to verify that the users you want to have access are assigned to the group that you select.
+1. Assign a group or leave the **Everyone** default. Be sure to verify that the users you want to have access are assigned to the group that you select. For instructions on how to assign the application to individual users and groups, see the [Assign applications for people and groups](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page-assign) topic in the Okta product documentation.
```

Comment:
... how to assign the app integration to individual users ...

[I know the topic title says "Assign applications for people and groups" but that is outmoded and will be updated when the target topic is updated.]

Reply:
Fixed